doc/api/tty.md

```diff
@@ -2,13 +2,19 @@
 
     Stability: 2 - Stable
 
-The `tty` module houses the `tty.ReadStream` and `tty.WriteStream` classes. In
-most cases, you will not need to use this module directly.
+The `tty` modules provides the `tty.ReadStream` and `tty.WriteStream` classes.
+In most cases, it will not be necessary to use this module directly. However,
+it can be accessed using:
 
-When Node.js detects that it is being run inside a TTY context, then `process.stdin`
-will be a `tty.ReadStream` instance and `process.stdout` will be
-a `tty.WriteStream` instance. The preferred way to check if Node.js is being run
-in a TTY context is to check `process.stdout.isTTY`:
+```js
+const tty = require('tty');
+```
+
+When Node.js detects that it is being run inside a TTY context, the 
```

Comment:
We should add a note about what a TTY is, because most people don't know what a TTY is, and the term is not very descriptive.

(i.e. "terminals")